components/snackbar/styles.module.scss

```
@@ -0,0 +1,113 @@
+@import '@/styles/index.scss';
+
+.snackbar {
+  @include Flex(null, null, center);
+  padding: 15px 20px;
+  border-radius: 10px;
+
+  .icon {
+    margin-right: 15px;
+  }
+
+  &.info {
+    background-color: $color-grayscale-50;
+    border: 1px solid $color-grayscale-200;
+    box-shadow: 0px 4px 20px 0px #0000000d;
+
+    &.small {
+      @include Font(
+        $font-size-xs,
+        $line-height-xs-18,
+        $font-weight-xs-regular,
+        $color-grayscale-500
+      );
+
+      .icon {
+        width: 15px;
+        height: 15px;
+      }
+    }
+
+    &.large {
+      @include Font(
+        $font-size-md,
+        $line-height-md,
+        $font-weight-md-regular,
+        $color-grayscale-500
+      );
+
+      .icon {
+        width: 20px;
+        height: 20px;
+      }
+    }
+  }
+
+  &.success {
+    background-color: $color-primary-green-100;
+    border: 1px solid $color-primary-green-200;
+    box-shadow: 0px 4px 20px 0px #0000000d;
+
+    &.small {
+      @include Font(
+        $font-size-xs,
+        $line-height-xs-20,
+        $font-weight-xs-semibold,
+        $color-primary-green-300
+      );
+
+      .icon {
+        width: 15px;
+        height: 15px;
+      }
+    }
+
+    &.large {
+      @include Font(
+        $font-size-md,
+        $line-height-md,
+        $font-weight-md-regular,
```

```suggestion
        $font-weight-md-semibold,
```
앗 제가 지난 코멘트에 말로만 설명드리고 코드에 표시를 안했었네요ㅠㅠ
코멘트 이렇게 작성하면 코드 깃헙에서도 바로 수정 가능한가요??